### PR TITLE
Fixes for running deterministic runs.

### PR DESCRIPTION
**Pull request by hu5970**

### CODE REVIEW

File: workflow/rocoto_funcs/mpassit.py [hunk 1]
```diff
@@ -73,7 +77,7 @@ def mpassit(xmlFile, expdir, do_ensemble=False):
   dependencies=f'''
   <dependency>
   <and>{timedep}
-  <datadep age="00:05:00"><cyclestr>&DATAROOT;/&NET;/&rrfs_ver;/&RUN;.@Y@m@d/@H{ensindexstr}/{wgf}/&RUN;_fcst_@H/</cyclestr><cyclestr offset="#fhr2#:00:00">diag.@Y-@m-@d_@H.@M.@S.nc</cyclestr></datadep>
+  <datadep age="00:05:00"><cyclestr>&DATAROOT;/&NET;/&rrfs_ver;/&RUN;.@Y@m@d/@H{ensindexstr}/{wgf}/&RUN;{wgf}_fcst_@H/</cyclestr><cyclestr offset="#fhr2#:00:00">diag.@Y-@m-@d_@H.@M.@S.nc</cyclestr></datadep>
```
Comment (chunhuazhou) [inline workflow/rocoto_funcs/mpassit.py]: Can you change {ensindexstr} to {ensdirstr}? Thanks!

File: jobs/JRRFS_UPP [hunk 2]
```diff
@@ -26,11 +26,15 @@ export HISTORY_INTERVAL=${HISTORY_INTERVAL:-1}
...
-  export RUN="rrfs"
+  export MEMID=""
+  export MEMDIR=""
 else
-  export RUN="ensrrfs"
+  export MEMID="_m${ENS_INDEX}"
+  export MEMDIR="/m${ENS_INDEX}"
 fi
+
 export jobid=${jobid:-${RUN}_upp_g${GROUP_INDEX}_${cyc}}
```
Comment (chunhuazhou) [inline jobs/JRRFS_UPP]: can you please change this to have "MEMID" information, similar to other scripts? 
export jobid=${jobid:-${RUN}_upp${MEMID}_g${GROUP_INDEX}_${cyc}}

File: jobs/JRRFS_MPASSIT [hunk 3]
```diff
@@ -26,10 +26,13 @@ export HISTORY_INTERVAL=${HISTORY_INTERVAL:-1}
...
 if [[ -z "${ENS_INDEX}" ]]; then
-  export RUN="rrfs"
+  export MEMID=""
+  export MEMDIR=""
 else
-  export RUN="ensrrfs"
+  export MEMID="_m${ENS_INDEX}"
+  export MEMDIR="/m${ENS_INDEX}"
 fi
 export jobid=${jobid:-${RUN}_mpassit_g${GROUP_INDEX}_${cyc}}
```
Comment (chunhuazhou) [inline jobs/JRRFS_MPASSIT]: can you please change this to have "MEMID" information, similar to other scripts?
export jobid=${jobid:-${RUN}_mpassit${MEMID}g${GROUP_INDEX}${cyc}}

File: jobs/JRRFS_UPP (hunk 2 above)
Comment (hu5970) [reply jobs/JRRFS_UPP]: Done

File: workflow/rocoto_funcs/mpassit.py (hunk 1 above)
Comment (hu5970) [reply workflow/rocoto_funcs/mpassit.py]: Done

File: jobs/JRRFS_MPASSIT (hunk 3 above)
Comment (hu5970) [reply jobs/JRRFS_MPASSIT]: Done

**Review by chunhuazhou — APPROVED** [review]

**Pull request merged**